lua/easy-dotnet/options.lua

```diff
@@ -138,16 +138,29 @@ local M = {
     },
     csproj_mappings = true,
     fsproj_mappings = true,
-    auto_bootstrap_namespace = true,
-    file_scoped = true,
+    auto_bootstrap_namespace = {
+      type = "file_scoped",
+      enabled = true
+    },
   },
 }
 
 local function merge_tables(default_options, user_options)
   return vim.tbl_deep_extend("keep", user_options, default_options)
 end
 
+--- Auto_bootstrap namespace can be either true or table with config
+local function handle_auto_bootstrap_namespace(a)
+  if type(a.auto_bootstrap_namespace) ~= "table" then
+    a.auto_bootstrap_namespace = {
+      type = "file_scoped",

```

Comment:
should probably keep "block_scoped" as default as it was the only option before?

Reply:
Agreed